docsite/rst/intro_configuration.rst

```diff
@@ -353,6 +353,25 @@ This option can be useful for those wishing to save fact gathering time. Both 's
 
     gathering = smart
 
+.. versionadded:: 2.1
+
+You can specify a subset of gathered facts using the following options:
+
```

Comment:
Still needs a double colon:

```
You can specify a subset of gathered facts using the following options::
```

That tells restructured text that the next indented text is a preformatted block.